### PR TITLE
Pivotal tracker

### PR DESCRIPTION
**Pull request by epaulet**

### CODE REVIEW

**Comment by epaulet** [comment]
@jipiboily Can you take a quick look please?

**Comment by epaulet** [comment]
Alright, NOW it's ready!

File: lib/integrations/pivotal_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      if response.code == 404
+        raise Integrations::UserConfigurationError.new('The project ID provided is was not found.')
+      elsif response.code == 403
+        raise Integrations::UserConfigurationError.new('The authorization token is invalid.')
+      elsif response.code != 200
+        raise Integrations::MisconfiguredIntegrationError.new('Invalid request to the Pivotal Tracker API.')
+      end
+    end
+
+    def message_color
```
Comment (jipiboily) [inline lib/integrations/pivotal_tracker.rb]: This was copied from the Slack integration and has no reason to be here :)

File: spec/lib/integrations/pivotal_tracker_spec.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        ]
+      }
+    end
+    let(:settings) do
+      {
+        project_id: 1141528,
+        auth_token: "8537292c903ca580bcd10b800709a136",
+      }
+    end
+    let(:expected_message) { "Your Rainforest Run <http://www.rainforestqa.com/ | Run #9: rainforest run> failed. Time to finish: 12 minutes 30 seconds" }
```
Comment (jipiboily) [inline spec/lib/integrations/pivotal_tracker_spec.rb]: The format does not make much sense here, as it's the format for Slack links. I think the format will have to be different on a per integration basis, or we could also define a `url_href` method in the base, with a typical HTML tag, that we could override in the Slack integration, and any other with custom tags. What do you think?

**Comment by jipiboily** [comment]
Other than those comments, looks good!

File: spec/lib/integrations/pivotal_tracker_spec.rb (hunk 2 above)
Comment (epaulet) [inline spec/lib/integrations/pivotal_tracker_spec.rb]: Makes sense.

Comment (epaulet) [inline spec/lib/integrations/pivotal_tracker_spec.rb]: Since this will change all of the integrations, should it be a different PR?

Comment (jipiboily) [inline spec/lib/integrations/pivotal_tracker_spec.rb]: Possibly yeah, either way works for me as this is unreleased code anyways for now.

**Comment by epaulet** [comment]
Merging this now then changing the hyperlinks in a separate PR
